src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -11,7 +11,7 @@
     <ArchiveName>dotnet-crossgen2</ArchiveName>
     <SharedFrameworkHostFileNameOverride>crossgen2</SharedFrameworkHostFileNameOverride>
     <!-- Build this pack for any RID if building from source. Otherwise, only build select RIDs. -->
-    <RuntimeIdentifiers Condition="'$(DotNetBuildSourceOnly)' != 'true'">linux-x64;linux-musl-x64;linux-arm;linux-musl-arm;linux-arm64;linux-musl-arm64;osx-x64;osx-arm64;win-x64;win-x86;win-arm64</RuntimeIdentifiers>
+    <RuntimeIdentifiers Condition="'$(DotNetBuildSourceOnly)' != 'true'">linux-x64;linux-musl-x64;linux-arm;linux-musl-arm;linux-arm64;linux-musl-arm64;linux-riscv64;linux-musl-riscv64;osx-x64;osx-arm64;win-x64;win-x86;win-arm64</RuntimeIdentifiers>
```

Comment:
this needs to be moved below and conditioned on `TargetArchitecture` like the FreeBSD case since we don't produce these in our official builds

Reply:
Do we produce all the others? linux-arm and linux-musl-arm were added _after_ my change in dotnet/installer, and it will be picked up in preview2.